Models.py

```diff
@@ -62,7 +63,7 @@ def from_json(cls, json, graph, push_updates=False):
 
     def update_to(self, graph, other_person):
         self.display_name = other_person.display_name
-        self.email = other_person.email
+        self.email = other_person.email.lower()
```

Comment:
I don't think we need this here. Since anytime we create a person we already set self.email = email.lower(), this should never be capitalized. LMK if I'm wrong.